its/pom.xml

```diff
@@ -29,6 +29,15 @@
     <jdk.min.version>11</jdk.min.version>
     <version.jacoco.plugin>0.8.8</version.jacoco.plugin>
   </properties>
+  
+  <dependencies>
+   <dependency>
+      <groupId>com.google.protobuf</groupId>
+      <artifactId>protobuf-java</artifactId>
```

Comment:
On `org.sonarlint.eclipse.its.OpenInIdeTest#getFirstIssue(...)` and `org.sonarlint.eclipse.its.OpenInIdeTest#getFirstBranch()`, I use classes from the package `org.sonarqube.ws.*`

Without the import, it failed even to compile because the dependency is required transitively. So the issues, I think, are coming from the orchestrator relying on the _protobuf_ dependency; maybe that can be changed on the orchestrator side?